illumio.json

```diff
@@ -1,249 +1,399 @@
-            "description": "Illumio PCE base URL",
-            "data_type": "string",
+        "hostname": {
             "required": true,
-            "value_list": [],
-            "default": "",
+            "data_type": "string",
+            "description": "Hostname",
             "order": 0
         },
-        "api_version": {
-            "description": "Illumio Core API version",
-            "data_type": "string",
+        "port": {
             "required": true,
-            "value_list": [],
-            "default": "v2",
+            "data_type": "numeric",
+            "description": "Port",
             "order": 1
         },
-        "org_id": {
-            "description": "Organization identifier",
-            "data_type": "numeric",
+        "api_key": {
             "required": true,
-            "value_list": [],
-            "default": 1,
+            "data_type": "string",
+            "description": "API key",
             "order": 2
         },
-        "api_user": {
-            "description": "Illumio Core API username",
-            "data_type": "password",
+        "api_secret": {
             "required": true,
+            "data_type": "password",
+            "description": "API Secret",
             "order": 3
         },
-        "api_key": {
-            "description": "Illumio Core API key",
-            "data_type": "password",
+        "org_id": {
             "required": true,
+            "data_type": "numeric",
+            "description": "Org ID",
             "order": 4
-        },
-        "verify_server_cert": {
-            "description": "Verify server certificate",
-            "data_type": "boolean",
-            "required": true,
-            "default": true,
-            "order": 5
         }
     },
     "actions": [
         {
             "action": "test connectivity",
             "identifier": "test_connectivity",
             "description": "Validate the asset configuration for connectivity using supplied configuration",
-            "verbose": "",
             "type": "test",
-            "read_only": true,
             "parameters": {},
             "output": [],
+            "read_only": true,
             "versions": "EQ(*)"
         },
         {
-            "action": "create virtual service",
-            "identifier": "create_virtual_service",
-            "description": "Create virtual service",
-            "verbose": "Create a virtual service with the given parameters.",
-            "type": "generic",
-            "read_only": false,
+            "action": "get traffic analysis",
+            "identifier": "get_traffic_analysis",
+            "description": "List of traffic on specified port",
+            "verbose": "The end_time will be the current time if a user provides no input and the start_time will be 5 years back from end_time if the user provides no input. Alllowed values for 'policy_decisions' parameter are: allowed, unknown, potentially_blocked, and blocked.",
```

Comment:
This description should be updated - I don't think we have these default values.

Reply:
Updated the description.